Procfile.dev

```diff
@@ -1,4 +1,5 @@
-js: yarn build:js --watch
+find-js: yarn build:js:find --watch
+publish-js: yarn build:js:publish --watch
```

Comment:
Should we swap these to `js-find` and `js-publish` to go from most common to least common?

It also aligns with the script names.